7.html

```diff
@@ -7,6 +7,20 @@
 <html>
 	<head>
 		<title>KindOf&lt;Polymorphism&gt;</title>
+		<meta name=”description” content=”In this talk Daniele Campogiani (https://twitter.com/dcampogiani) will explore the tagless final approach abstracting over different effects in Kotlin.
+ 
+ Suppo”>
```

Comment:
Suppo?

Reply:
Suppo!
https://github.com/fp-in-bo/fp-in-bo.github.io/pull/5/files#diff-205ed59a9b5856b658e1342ca7777ddaR130

take(160)

Reply:
🤷‍♂️